pandas/core/format.py

```diff
@@ -129,45 +129,62 @@ def to_string(self):
 
 class SeriesFormatter(object):
 
-    def __init__(self, series, buf=None, header=True, length=True,
-                 na_rep='NaN', name=False, float_format=None, dtype=True):
+    def __init__(self, series, buf=None, length=True,
+                 na_rep='NaN', name=False, float_format=None, dtype=True,
+                 max_rows=None):
         self.series = series
         self.buf = buf if buf is not None else StringIO()
         self.name = name
         self.na_rep = na_rep
         self.length = length
-        self.header = header
```

Comment:
header in this case of series only concerns the index name i think. I removed `header` because it was unused in SeriesFormatter previously and it was also not accessible through the Series.to_string. But I'm ok to add it back. That should be quick.